docs/pages/installation/A_Installation-Ubuntu.md

```diff
@@ -1,12 +1,24 @@
-## Installing on Ubuntu 14.04, 16.04, (experimental: 18.04)
+## Installing on Ubuntu 
+14.04 (depricated), 16.04, 18.04, 20.04 (experimental)
+### Automatic installation
+For your convenience, we provide an install script to automate the whole installtion. Please make sure that you have an [SSH key activated on your GitHub account.](https://help.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
+To install maplab, use the following commands
+```bash
+wget https://raw.githubusercontent.com/ethz-asl/maplab/docs/install-instructions/install.sh
+sudo bash install.sh # Use the argument "experimental" to also install our latest features if your are eligible.
+```
 
-### Install required system packages
+### Manual installation
+If you prefer manually installing all compoments, following this guideline.
 
-```bash
-# Install ROS 
-export UBUNTU_VERSION=xenial #(Ubuntu 16.04: xenial, Ubuntu 14.04: trusty, Ubuntu 18.04: bionic)
-export ROS_VERSION=kinetic #(Ubuntu 16.04: kinetic, Ubuntu 14.04: indigo, Ubuntu 18.04: melodic)
+```bash 
+export UBUNTU_VERSION=$(lsb_release -cs) #(Ubuntu 16.04: xenial, Ubuntu 14.04: trusty, Ubuntu 18.04: bionic)
+export ROS_VERSION=melodic #(Ubuntu 16.04: kinetic, Ubuntu 14.04: indigo, Ubuntu 18.04: melodic)
```

Comment:
Maybe leave the numeric order in the parenthesis or if 14.04 is the least suggested alternative maybe place it at the end instead of having it in the middle.